.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  # Enable daily version updates for pip
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    # limit to security updates only
+    open-pull-requests-limit: 0
```

Comment:
`open-pull-requests-limit: 0` is what controls security updates only?

Reply:
Apparently so, that's how the docs explain it here:
https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/customizing-dependabot-security-prs#about-customizing-pull-requests-for-security-updates

> If you want the customization for a package ecosystem to only apply to security updates (and exclude version updates), set the open-pull-requests-limit key to 0.